### DISCUSSION TITLE
Making a permanent connection to a peer

### DISCUSSION THREAD

**Discussion by Jacek-ghub** [Chia-Network/chia-blockchain#10451] [Ideas]
Could a permanent flag be added to "CONNECT TO OTHER PEERS" dialog.  Peers that would be marked as such would be potentially the last to drop, and the first to reconnect.  I was running a couple of nodes at my location, where one was syncing from scratch, and connection between those two nodes were constantly dropped.

This feature would be helpful for those of us who for instance run something like master / slave full node setup at one location to have a backup blockchain db source.  Of course, in such setup, the ideal implementation would be that the syncing node would primarily use that other box to reduce the overall chia network traffic (say one out of N blocks would be downloaded from remote peers, to make sure that db is not poisoned - the current download implementation is a round-robin process, so that would be a preferential round-robin).

If that would be considered not secure (e.g., someone would like to have just those manual connections), even having a limit of one such connection would be good enough.

**Comment by mariano54** [comment]
Yes this is a good idea, we do have a trusted peers list, we need to make this configurable and allow adding this from the GUI.

**Comment by Jacek-ghub** [comment]
Thank you.  

I used UI as an example.  Whether it would be exposed through UI or CLI doesn't really make a difference but would really help those that are looking for something like master / slave db backups.

Also, if sharing ca folders would be needed, then again, I don't see that as an issue.  Whatever makes it more secure is always good to have.
